### ISSUE TITLE
Training loss is nan in char-rnn example as rendered at Pytorch site

### ISSUE THREAD

**Issue by vladserkoff** [pytorch/tutorials#847]
Apparently training has failed when running the code to render tutorial for Pytorch site.
Have a look at the training section and below https://pytorch.org/tutorials/intermediate/char_rnn_classification_tutorial.html
```
5000 5% (0m 12s) 2.4177 Coelho / Portuguese ✓
10000 10% (0m 21s) 2.5862 Desrochers / Dutch ✗ (French)
15000 15% (0m 30s) 1.1451 Minitsky / Polish ✗ (Russian)
20000 20% (0m 38s) 2.3326 Koza / Japanese ✗ (Czech)
25000 25% (0m 47s) 1.5165 Mateus / Arabic ✗ (Portuguese)
30000 30% (0m 56s) 2.5533 Duchamps / Greek ✗ (French)
35000 35% (1m 4s) nan Nam / Russian ✗ (Korean)
40000 40% (1m 13s) nan Assaf / Russian ✗ (Arabic)
45000 45% (1m 22s) nan Ko / Russian ✗ (Korean)
50000 50% (1m 30s) nan Pho / Russian ✗ (Vietnamese)
...
```

**Comment by vladserkoff** [comment]
fixed
